src/main/java/seedu/toluist/controller/TaskController.java

```diff
@@ -110,4 +110,8 @@ private CommandResult delete(TodoList todoList, Task task) {
         todoList.remove(task);
         return new CommandResult(DELETE_TASK_RESULT);
     }
+
+    public static String[] getCommandWords() {
+        return new String[] { UPDATE_TASK_COMMAND, DELETE_TASK_COMMAND, ADD_TASK_COMMAND };
```

Comment:
COMMAND_WORD
UPDATE_TASK_COMMAND, DELETE_TASK_COMMAND, ADD_TASK_COMMAND
these two lines are not consistent with each other (on where the COMMAND is in)